elasticsearch_opentracing/__init__.py

```diff
@@ -22,25 +24,12 @@ def enable_tracing():
 def disable_tracing():
     tls.tracing_enabled = False
 
-def get_active_span():
```

Comment:
I'd prefer if we kept these functions, although deprecated - and relying on `g_tracer`, i.e.

```python
    return g_tracer.active_span
```

This way we could make the life of current users easier (and help them with the migration ;) )

Reply:
These are helpful functions, so removing them is a bit overzealous.  Added deprecation warnings.